seonshine_frontend/src/components/organims/sideBar/index.tsx

```diff
@@ -55,7 +89,16 @@ const Sidebar = ({ role }: SidebarPropsType) => {
           aria-label="notifications"
           className="w-12 h-12"
         >
-          <Avatar className="bg-gray-200" />
+          <Avatar
+            className="bg-gray-200"
+            onClick={handleOpenModal}
+          />
+          <UserProfileModal
```

Comment:
using `{isModalOpen && <UserProfileModal}`. It will cause the UserProfile to remain in state